Formula/create-api.rb

```diff
@@ -0,0 +1,34 @@
+class CreateApi < Formula
+  desc "Delightful code generator for OpenAPI specs"
+  homepage "https://github.com/CreateAPI/CreateAPI"
+  url "https://github.com/CreateAPI/CreateAPI/archive/refs/tags/0.0.5.tar.gz"
+  sha256 "c250ff140af83e093d86fef0dd18b87363ae91a087c0804bedb40f9922989093"
+  license "MIT"
+  head "https://github.com/CreateAPI/CreateAPI.git", branch: "main"
+
+  depends_on xcode: "13.0"
+
+  uses_from_macos "swift"
+
+  resource "homebrew-test-data" do
+    url "https://github.com/CreateAPI/CreateAPI/archive/refs/tags/0.0.5.tar.gz"
+    sha256 "c250ff140af83e093d86fef0dd18b87363ae91a087c0804bedb40f9922989093"
+  end
```

Comment:
Do we need an exact copy of the original tarball for this?

Reply:
I didn't think much about it, but no we do not. I updated the resource to download the schema directly from raw.githubusercontent.com instead. It simplified things a lot, thanks!